src/custom/components/Settings/SettingsMod.tsx

```diff
@@ -43,18 +42,6 @@ export const StyledMenuIcon = styled(Settings)`
   }
 `
 
-const StyledCloseIcon = styled(X)`
-  height: 20px;
-  width: 20px;
-  :hover {
-    cursor: pointer;
-  }
-
-  > * {
-    stroke: ${({ theme }) => theme.text1};
-  }
-`
-
```

Comment:
Remember we want to keep mod files close to the original.
So, if we don't need it and it hasn't been added here, we just comment it out rather than removing.

Reply:
Thank you! Fixed